src/frontend/frontend/views/story_bookmark_views.py

```
@@ -0,0 +1,241 @@
+
+    @classmethod
+    def _response_status(cls, response) -> int:
+        return getattr(response, "status_code", 500) or 500
+
+    @classmethod
+    def _invalidate_bookmark_cache(cls, bookmark_id: str | None = None) -> None:
+        try:
+            DataPersistenceLayer().invalidate_model_cache_locally(StoryBookmark, bookmark_id)
+        except HTTPException:
+            raise
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)